readme.md

```diff
@@ -116,6 +116,20 @@ NOTE: specifying an output path with multiple formats ignores your specified ext
 * `./<HOST>_<DATE>.report.html`
 * `./<HOST>_<DATE>.artifacts.log`
 
+### Testing on a site with authentication
+
+If you have installed Lighthouse globally 
+
+ - `chrome-debug`
+ - open and login to your site
+ - `lighthouse http://mysite.com`
+ 
+If you use repository
```

Comment:
👍
I was thinking about users who doesn't know about it, but you are right, if person using repo he/she should know about this kind of stuff.
nuking... 💥